V3DemoProcfile

```diff
@@ -1,5 +1,5 @@
 # Use goreman to run `go get github.com/mattn/goreman`
-# etcd1 is the default client server for etcdctlv3 commands
+# etcd1 is the default client server for etcdctl commands
```

Comment:
I think default endpoints are all of these 3. https://github.com/coreos/etcd/blob/master/etcdctlv3/main.go#L48

Maybe just remove this line?